exercises/diamond/test/Tests.hs

```diff
@@ -1,17 +1,62 @@
 {-# LANGUAGE RecordWildCards #-}
 
-import Data.Foldable     (for_)
-import Test.Hspec        (Spec, describe, it, shouldBe)
-import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+import Data.Foldable       (for_)
+import Test.Hspec          (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner   (configFastFail, defaultConfig, hspecWith)
```

Comment:
it doesn't seem necessary to increase the indentation. no `import` being used here is longer than `Test.Hspec.Runner`, is it?